descriptors/ModuleDescriptor-template.json

```diff
@@ -82,6 +82,46 @@
             "inventory-storage.service-points.item.get"
           ]
         },
+        {
+          "methods": ["DELETE"],
+          "pathPattern": "/bl-users/by-id/{id}",
+          "permissionsRequired": ["users-bl.item.delete"],
+          "modulePermissions": [
+            "users.item.get",
+            "users.item.delete",
+            "circulation-storage.loans.collection.get",
+            "circulation-storage.requests.collection.get",
+            "accounts.collection.get",
+            "manualblocks.collection.get",
+            "proxiesfor.collection.get"
+          ]
+        },
+        {
+          "methods": ["GET"],
+          "pathPattern": "/bl-users/by-id/{id}/open-transactions",
+          "permissionsRequired": ["users-bl.transactions.get"],
+          "modulePermissions": [
+            "users.item.get",
+            "circulation-storage.loans.collection.get",
+            "circulation-storage.requests.collection.get",
+            "accounts.collection.get",
+            "manualblocks.collection.get",
```

Comment:
The use of `manualblocks.collection.get` suggests that this endpoint depends upon a manual blocks API, and yet there seems to be no dependency for this in the `requires` section, is that intentional?

Reply:
`manualblocks.collection.get` is defined by mod-feesfines. It is required for the `/manualblocks` endpoint that is provided by the interface `feesfines` `v16.3`. `feesfines` `v16.3` is already listed in the `requires` section.

Reply:
Damn, thank you. I find it confusing that patron blocks are not implemented by the patron blocks module or in an interface that refers to patron blocks :-/